File: C#/InvertBinaryTree.cs
using System;

namespace C_{

    class TreeNode {
    int val;
    public TreeNode left;
    public TreeNode right;
    TreeNode() {}
    TreeNode(int val) { this.val = val; }
    TreeNode(int val, TreeNode left, TreeNode right) {
          this.val = val;
          this.left = left;
          this.right = right;
    }
}
    class InvertBinaryTree{
        public TreeNode InvertTree(TreeNode root) {
            if (root == null) 
                return null;
            TreeNode leftNode = InvertTree(root.left);
            TreeNode rightNode = InvertTree(root.right);
            root.left = rightNode;
            root.right = leftNode;
            return root;
        }
    }
}

File: Python/InvertBinaryTree.py
class TreeNode:
    def __init__(self, val=0, left=None, right=None):
        self.val = val
        self.left = left
        self.right = right

class InvertBinaryTree:
    def invertTree(self, root: TreeNode) -> TreeNode:
        if not root: return None
        root.left, root.right = self.invertTree(root.right), self.invertTree(root.left)
        return root